src/app/registration/facility-select/facility-select.component.ts

```diff
@@ -261,6 +272,21 @@ export class FacilitySelectComponent implements OnInit {
     return DateTime.now().setZone('America/Vancouver');
   }
 
+  async getSystemTime(){
+      const facilityWithCurrentTime = this.facilities.find(facility => facility.currentTime);
+      this.systemTime = facilityWithCurrentTime ? facilityWithCurrentTime.currentTime : null;
+      this.systemTimePST = this.convertUTCToPST(this.systemTime);
+      if (!this.isOpeningHourPast){
+        this.validBookingTime = false;
+      } 
+  }
+
+  convertUTCToPST(utcTime){
```

Comment:
spacing

Reply:
Resolved in [Commit43beab5](https://github.com/bcgov/parks-reso-public/pull/461/commits/43beab52a1c7d818910d777c2133aefa47ce60ff)